e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/add-to-an-existing-payment.spec.js

```diff
@@ -85,7 +88,14 @@ context(`${PDC_TEAMS.PDC_RECONCILE} users can add fee records to existing paymen
     pages.utilisationReportAddToAnExistingPaymentPage
       .availablePaymentGroups()
       .should('contain', 'There is one existing payment that the reported fees will be added to');
-    pages.utilisationReportAddToAnExistingPaymentPage.availablePaymentGroups().should('contain', 'GBP 60.00');
+    pages.utilisationReportAddToAnExistingPaymentPage.availablePaymentGroups().should('contain', 'GBP 450.00');
     pages.utilisationReportAddToAnExistingPaymentPage.availablePaymentGroups().should('contain', 'Payment reference: REF01234');
   });
+
+  it('should add the selected fees to the payments and update the fee record status after clicking the continue button', () => {
+    pages.utilisationReportAddToAnExistingPaymentPage.continueButton().click();
+
+    pages.utilisationReportPage.premiumPaymentsTab.getPaymentLink(PAYMENT_ID_ONE).should('contain', 'GBP 450.00');
+    pages.utilisationReportPage.premiumPaymentsTab.getPremiumPaymentsTableRow(FEE_RECORD_ID_ONE).should('contain', 'MATCH');
+  });
```

Comment:
Aware this is a bit counter to some of the existing tests, but we should be looking to combine e2e-tests into testing a journey as a whole where possible, given the before each does quite a lot and therefore takes a bit of time. Could we perhaps combine this with the above two tests to have one test that covers all the checking of things showing on the page and then at the end clicks continue?

Also how much effort would it be to add a second test which has multiple payment groups to test the case with radio button?

Reply:
Yep, combined these now thanks. Added two additional test cases too:
- `should automatically select the first payment group when there is only one group to choose from`
- `should display an error message when there are multiple payments to choose from and none have been selected`